cli/src/main/java/org/jboss/galleon/cli/HelpSupport.java

```diff
@@ -168,6 +89,163 @@ private static String pad(int max, String k) {
+            });
+
+            for (CommandLineParser<? extends Command> childParser : allChildParsers) {
+                ProcessedCommand<? extends Command> childProcessedCommand = childParser.getProcessedCommand();
+                sb.append(Config.getLineSeparator()).append("    ");
+                sb.append(childProcessedCommand.name());
+                if (childProcessedCommand.hasArguments() || childProcessedCommand.hasArgument()) {
+                    sb.append(" <arg>");
+                }
+                sb.append(" : ").append(trimDescription(childProcessedCommand.description()));
+                sb.append(getCommandOptions(childProcessedCommand, 4));
+            }
+        }
+
+        return sb.toString();
+    }
+
+    private static String getHelpFooter() {
+        try {
+            return getHelp(getHelpPath("help-command.txt"));
+        } catch (IOException e) {
+            return "Failed to read help-command.txt. " + e.getLocalizedMessage();
+        }
+    }
+
+    private static String getCommandOptions(ProcessedCommand<? extends Command> command, int offset) {
+        StringBuilder sb = new StringBuilder();
+        if (command.hasOptions()) {
+            TreeMap<String, String> orderedOptions = new TreeMap<>();
+            sb.append(Config.getLineSeparator());
+
+            if (command.hasArgument() || command.hasArguments()) {
+                processArguments(command, offset, sb);
+                // if the command has an empty or null argument we don't add a new line
+                if (sb.lastIndexOf(Config.getLineSeparator()) != sb.length() - 1) {
+                    sb.append(Config.getLineSeparator());
+                }
+            }
+
+            List<ProcessedOption> options = command.getOptions();
+            for (ProcessedOption option : options) {
+                orderedOptions.put(option.name(), trimDescription(
+                        option.getFormattedOption(offset + TAB.length(), 0, 80)));
+            }
+
+            for (Map.Entry<String, String> option : orderedOptions.entrySet()) {
+                sb.append(option.getValue());
+                if (!option.equals(orderedOptions.lastEntry())) {
+                    sb.append(Config.getLineSeparator());
+                }
+            }
+        } else if (command.hasArguments() || command.hasArgument()) {
+            sb.append(Config.getLineSeparator());
+            processArguments(command, offset, sb);
+        }
+
+        return sb.toString();
+    }
+
+    private static void processArguments(ProcessedCommand<? extends Command> command, int offset, StringBuilder sb) {
+        if (command.hasArgument()) {
+            if (command.getArgument().description() != null && !("".equals(command.getArgument().description()))) {
+                handleOffset(offset, sb);
+                sb.append(TAB).append("<arg> : ").append(trimDescription(command.getArgument().description()));
```

Comment:
The ":" after <arg> is not required. --<option> have no such separator. Makes the output a bit inconsistent.